### ISSUE TITLE
add intro video on Tour page

### ISSUE THREAD

**Issue by mfc** [QubesOS/qubes-issues#1878]
https://www.youtube.com/watch?v=dD0_gq_ugw8

We have already received author permission to put it on the website. Just want to make sure we do it because it is a great demo/tour of Qubes.

**Comment by bnvk** [comment]
@mfc @marmarek @rootkovska @axon-qubes i've pushed an [implementation](https://www.qubes-os.org/tour/) of the video on the Tour page. I'm currently just using the YouTube video embedded and have some chapter splitting controlled by JavaScript.

Perhaps the largest negative I see with this approach, is that embedding YouTube videos also loads numerous Google + YouTube trackers... it's not a huge issue for TorBB users, but something to consider. Alternatively, we could self host the video (about 150 MB or larger) somewhere to eliminate such trackers. Additionally, the later approach would work better for Offline Documentation as we could include the video with the Docs.

**Comment by axon-qubes** [comment]
I like the idea of having the video on the page. However, the pros and cons of self-hosting vs. embedding from YouTube seem to largely cancel each other out, so I'm not sure which is better.

Some thoughts and suggestions on the current implementations:
 * The box in which the video is embedded looks a bit too wide.
 * The list of links to different times in the video is very helpful, but it takes up quite a bit of space. It would be nice to make it more compact, since right now it looks like the only content on the page is the video.
 * It would be good to include some indication of what those links are for. At first, I thought they were links to related articles. Only after I hovered over them and saw timestamps was I able to infer that they must be links to specific times in the video.

**Comment by axon-qubes** [comment]
We should also consider adding at least some of the other videos linked to on the News page. Perhaps have a dedicated Videos page?

**Comment by mfc** [comment]
github can host large files now. Would the chapters / bookmarks still work? 

> We should also consider adding at least some of the other videos linked to on the News page. Perhaps have a dedicated Videos page?

those other videos are not appropriate for the Tour page, so yes you can consider a Media page or Video page, or just keep it as is. but that isn't really a topic for this ticket.

**Comment by andrewdavidwong** [comment]
FYI, a website visitor has complained about the video autoplaying (and thereby using up mobile data):

https://twitter.com/realnzall/status/728213010146267136

**Comment by mfc** [comment]
the intro video was added to the tour page so I am closing this ticket.

I still think we should explore hosting the video from github, but that can be another ticket.
